cpp/ql/test/examples/expressions/PrintAST.expected

```diff
@@ -335,20 +335,58 @@ Conversion4.c:
 #    2|         0: [VariableAccess] x
 #    2|             Type = [IntType] int
 #    2|             ValueCategory = lvalue
-#    2|         1: [ParenthesisExpr] (...)
+#    2|         1: [Literal] 7
+#    2|             Type = [IntType] int
+#    2|             Value = [Literal] 7
+#    2|             ValueCategory = prvalue
+#    2|         1 converted: [ParenthesisExpr] (...)
 #    2|             Type = [IntType] int
 #    2|             Value = [ParenthesisExpr] 7
 #    2|             ValueCategory = prvalue
-#    2|           expr: [CStyleCast] (int)...
+#    2|           : [CStyleCast] (int)...
```

Comment:
Here's a child with no label. Please remind me why there are sometimes no labels and what we can do about it.

Reply:
That was a bug, fixed now.